Automattic/wp-calypso
# Stepper Flow: Extract intent-screen to its own package

#### Changes proposed in this Pull Request

Extract intent-screen from https://github.com/Automattic/wp-calypso/tree/trunk/client/signup/steps/intent to its own package

#### Testing instructions



* Checkout the branch
* Run `yarn start`
* Check that the intent screen is working 
* Check that tests succeed (tests still to be added)


Intent screen:
URL: https://wordpress.com/start/setup-site/intent?siteSlug=Website&loading_ellipsis=1
![image](https://user-images.githubusercontent.com/52076348/154670300-96782bab-b10d-4a61-ad3f-5b079ca351fa.png)




Related to #60809
Fixes #60809
